docs/components/movement-sensor/cameramono.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Configure `camera_mono` for Visual Odometry"
+linkTitle: "camera_mono"
+weight: 10
+type: "docs"
+description: "Configure `camera_mono`, an experimental visual odometry model."
+# SMEs: Rand

```

Comment:
Hide this. It's not in index.

Reply:
Ah true, I left it out of the index page by mistake. We currently have it in the docs though; do we want it removed?

Reply:
Yes

Reply:
Per slack conversation I will add it to the index